adapter/thing_management.md

```diff
@@ -126,15 +126,15 @@ Every adapter should define its own service name.
-| name         | string    | `"out_bin_switch"`                               | Name of the service, as specified in [device services](/device_services/device_services.md) documentation.                                               |
-| address      | string    | `"/rt:dev/rn:zw/ad:1/sv:out_bin_switch/ad:21_0"` | An address of the service starting from a resource type segment. Service address segment should contain thing address and optional group identification. |
-| groups       | str_array | `["ch_0"]`                                       | Groups to which the service belongs. It is recommended to never attach service to more than a single group.                                              |
-| enabled      | bool      | `true`                                           | Indicates whether service is enabled or not.                                                                                                             |
-| interfaces   | array     | `[{"msg_t":"evt.binary.report"}]`                | An array of [`interface_spec`](#definitions) objects for all interfaces provided by the service.                                                         |
-| props        | any_map   | `{"sup_units": ["W"]}`                           | A map of service properties.                                                                                                                             |
-| prop_set_ref | string    | `"set_a"`                                        | Reference to a property set defined in `prop_set`(#definitions). These are technology specific and for informational purpose only.                       |
+| Field        | Type      | Example                                          | Description                                                                                                                                                                                                                       |
+|--------------|-----------|--------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| name         | string    | `"out_bin_switch"`                               | Name of the service, as specified in [device services](/device_services/device_services.md) documentation.                                                                                                                        |
+| address      | string    | `"/rt:dev/rn:zw/ad:1/sv:out_bin_switch/ad:21_0"` | An address of the service starting from a resource type segment. Service address segment must contain thing address and optional group identification. See [topic format](/fimp/topic_format.md#topic-segments) for more details. |
```

Comment:
must -> shall for mandatory requirement

Reply:
Well, it's apple or oranges. There are two universally accepted standards I think:

https://www.ietf.org/rfc/rfc2119.txt

https://medium.com/@mrandig/iso-9001-2015-verbal-forms-dc50988a5aec

I believe it's just a matter of personal preference to choose one.

Reply:
Zwave specification uses: https://www.ietf.org/rfc/rfc2119.txt
Just letting You know :D @boguslaw-wojcik

Reply:
Guys, I personally I don't care. 

For me in documentation: 
* shall / have to / must = required
* should = recommended
* might / may / can = optional

I see no reason to limit vocabulary to strict ISO.